app/src/main/java/net/gsantner/markor/format/zimwiki/ZimWikiHighlighter.java

```diff
@@ -0,0 +1,145 @@
+/*#######################################################
+ *
+ *   Maintained by Gregor Santner, 2018-
+ *   https://gsantner.net/
+ *
+ *   License of this file: Apache 2.0 (Commercial upon request)
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+#########################################################*/
+package net.gsantner.markor.format.zimwiki;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.text.InputFilter;
+import android.text.Spannable;
+
+import net.gsantner.markor.format.ListHandler;
+import net.gsantner.markor.model.Document;
+import net.gsantner.markor.ui.hleditor.Highlighter;
+import net.gsantner.markor.ui.hleditor.HighlightingEditor;
+import net.gsantner.markor.util.AppSettings;
+
+import java.util.regex.Pattern;
+
+import other.writeily.format.zimwiki.WrZimWikiHeaderSpanCreator;
+
+public class ZimWikiHighlighter extends Highlighter {
+    private final boolean _highlightBiggerHeadings;
+    public final String _fontType;
+    public final Integer _fontSize;
+
+    private static final int COLOR_HEADING = 0xff4e9a06;
+    private static final int HIGHLIGHT_BACKGROUND_COLOR_LIGHT_MODE = 0xffffff00;
+    private static final int HIGHLIGHT_BACKGROUND_COLOR_DARK_MODE = 0xffFFA062;
+    private static final int UNORDERED_LIST_BULLET_COLOR = 0xffdaa521;
+    private static final int ORDERED_LIST_NUMBER_COLOR = 0xffdaa521;
+    private static final int LINK_COLOR = 0xff1ea3fd;// 0xff0000ff;
```

Comment:
* ✔️  feedback - gs1

color for links not much readable. We already have a specific blue color for links that used in ALL formats (todo/plaintext/markdown), so just gonna swap it to it (taking care of it)

![c](https://user-images.githubusercontent.com/6735650/99146641-55a9f780-267a-11eb-8fb6-f8db3c99949c.png)

Reply:
I took care of it, just screenshot for reference